iterm/verify-install.sh

```diff
@@ -0,0 +1,3 @@
+
+test -d $HOME/Applications/"iTerm.app" ||
```

Comment:
have you tested this?  i think your line break might need to be escaped with `\`